src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int radius = random.nextInt(1,10);
+    private final int sideA = random.nextInt(1,10);
+    private final int sideB = random.nextInt(1,10);
+    private final int height = random.nextInt(1, 10);
+
+    public Figure getRandomFigure() {
+        Figure[] figure = new Figure[] {new Square(sideA,
```

Comment:
You need to create a switch/case
Where we will get the number by the number of figures. And create a separate figure for each number. We will also not get different shapes through the array.